assets/application.js

```diff
@@ -18,7 +18,14 @@ require('govuk-template')
 require('javascripts')
 require('./components')
 require('./components/show-hide-content/show-hide-content-init')
-var GOVUK = require('stageprompt')
 
-// initialise stageprompt for Analytics
-GOVUK.performance.stageprompt.setupForGoogleAnalytics()
+var stub = function() {console.log('This feature has been removed, please use Tracking Consent instead.')};
```

Comment:
Would that message provide enough context for service teams to know what to do?

Reply:
I've updated this, is the new messsage OK?